### PR TITLE
start the button page

### PR DESCRIPTION
**Pull request by CrowdHailer**

### CODE REVIEW

File: docs/did-button-branding/index.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+layout: "doc"
+title: "Branding DID.app buttons."
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:3]: Maybe try: 'Sign in with DID.app' buttons for your website

File: docs/did-button-branding/index.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+abstract: "DID.app provides several styles of icon that you can use to let people sign in."
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:4]:
```suggestion
abstract: "DID.app provides several button designs that you can use on your website to let users know they can sign in with DID.app"
```

File: docs/did-button-branding/index.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+<style>
+  .markdown-body img {
+    background: none;
+  }
+</style>
+
+You can use either our "D.I.D Logo" or "D Icon" to indicate to your users that they may sign in with DID.app.
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:13]:
```suggestion
You can use either our "DID Logo" or "D Icon" in your button.
```

File: docs/did-button-branding/index.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+Both come in four styles; **default**, **light**, **dark-mono** and **light-mono**.
+
+### The D.I.D Logo
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:16]:
```suggestion
### The DID.app Logo
```

File: docs/did-button-branding/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+```html
+<img src="/icon.svg" style="padding:0.5em;">
+<img src="/icon.svg#light" style="padding:0.5em;background:#2a454e;">
+<img src="/icon.svg#light-mono" style="padding:0.5em;background:#00dfc0;">
+<img src="/icon.svg#dark-mono" style="padding:0.5em;background:#00dfc0;">
+```
+
+### Creating buttons
+
+##### DID.app as a social login provider
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:50]:
```suggestion
##### Using DID.app buttons alongside other social sign in providers
```

File: docs/did-button-branding/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+When offering DID.app as one option among several login providers you **should** follow the guidance on this page.
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:52]:
```suggestion
When offering DID.app as one option among several sign in providers you **should** follow the guidance on this page.
```

File: docs/did-button-branding/index.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+</div>
+
+```html
+<img src="/icon.svg" style="padding:0.5em;">
+<img src="/icon.svg#light" style="padding:0.5em;background:#2a454e;">
+<img src="/icon.svg#light-mono" style="padding:0.5em;background:#00dfc0;">
+<img src="/icon.svg#dark-mono" style="padding:0.5em;background:#00dfc0;">
+```
+
+### Creating buttons
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:48]:
```suggestion
### When to use DID.app buttons
```

File: docs/did-button-branding/index.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+
+##### DID.app as a social login provider
+
+When offering DID.app as one option among several login providers you **should** follow the guidance on this page.
+If your authentication process is only handled by DID.app you may use the buttons show here, but are not required to.
+
+### Examples
+
+A light button that can be used on most pages.
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:57]:
```suggestion
A light button that can be used on most pages:
```

File: docs/did-button-branding/index.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+```html
+<div style="display:inline-block;padding:0.3em;border:1px solid gray;margin:1em;border-radius:4px;">
+  <span style="display:flex;align-items:center;">
+    <img src="/icon.svg" style="height:30px">
+    <span style="padding:0 1rem;">Sign in with DID.app</span>
+  </span>
+</div>
+```
+
+A block style sign in button
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:75]:
```suggestion
A block style sign in button:
```

File: docs/did-button-branding/index.md [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+```html
+<div style="display:inline-block;padding:0.3em;margin:1em;background:#00dfc0;color:white;">
+  <span style="display:flex;align-items:center;">
+    <img src="/logo.svg#light-mono" style="height:30px">
+    <span style="padding:0 1rem;font-weight:bold;white-space:nowrap;">Sign in</span>
+  </span>
+</div>
+```
+
+A dark sign in button
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:93]:
```suggestion
A dark sign in button:
```

File: docs/did-button-branding/index.md [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+```html
+<div style="display:inline-block;padding:0.3em;margin:1em;background:#2a454e;color:white;">
+  <span style="display:flex;align-items:center;">
+    <img src="/logo.svg#light" style="height:30px">
+    <span style="padding:0 1rem;font-weight:bold;white-space:nowrap;">Sign in</span>
+  </span>
+</div>
+```
+
+Icon only button.
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:111]:
```suggestion
Icon only button:
```

File: docs/did-button-branding/index.md [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+
+<img src="/icon.svg#light-mono" style="height:40px;background:#00dfc0;padding:0.5em;border-radius:1em;">
+
+```html
+<img src="/icon.svg#light-mono" style="height:40px;background:#00dfc0;padding:0.5em;border-radius:1em;">
+```
+
+Matching your own app's color pallet.
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:119]:
```suggestion
### Custom button design
```

File: docs/did-button-branding/index.md [hunk 13]
```diff
@@ -0,0 +1,133 @@
...
+
+When not using the DID.app green pallet, it is recommended to use the mono style icon/logo.
```
Comment (richardesigns) [inline docs/did-button-branding/index.md:121]:
```suggestion
If you would like to use the DID.app buttons within your own color scheme we recommend to use the mono style icon and logo:
```

File: docs/did-button-branding/index.md (hunk 1 above)
Comment (CrowdHailer) [reply docs/did-button-branding/index.md:3]: not sure. this is a part of the docs page so can be a bit more formal, also I copied that title off the sign in with google docs

File: docs/did-button-branding/index.md (hunk 3 above)
Comment (CrowdHailer) [reply docs/did-button-branding/index.md:13]: I was wondering about giving the "DID" logo a name to separate it from the "D logo". They're all DID.app logos so I thought D.I.D made it specific I was talking about the three letter version

File: docs/did-button-branding/index.md (hunk 7 above)
Comment (CrowdHailer) [reply docs/did-button-branding/index.md:48]: Going to make this "How to use DID.app buttons"

File: docs/did-button-branding/index.md (hunk 12 above)
Comment (CrowdHailer) [reply docs/did-button-branding/index.md:119]: https://developers.google.com/identity/branding-guidelines#matching

**Comment by CrowdHailer** [comment]
@richardesigns I want to remove "DID_logo_ident_mono_blue.svg" from the project, to make sure we always use the same logo. There's no where in the project using them. I think anywhere else we put them we uploaded copies, so there is no harm removing them.

Just a heads up incase you rely on them

File: docs/did-button-branding/index.md (hunk 3 above)
Comment (richardesigns) [reply docs/did-button-branding/index.md:13]: Makes sense, why not say 'You can use either our three letter 'DID' logo or single letter 'D' icon.

File: docs/did-button-branding/index.md (hunk 1 above)
Comment (richardesigns) [reply docs/did-button-branding/index.md:3]: ok :-)  I think it's fine.  first time I read it was like what do you mean branding DID.app buttons.

is it Branding. DID.app buttons..

as in subjject: BRanding.

Content: DID.app buttons.

Just felt weird.  

Not overly worried.  Time is of the essence, choose one, implement it.

File: docs/did-button-branding/index.md (hunk 4 above)
Comment (richardesigns) [reply docs/did-button-branding/index.md:16]: Yea, I think the DID logo and the D icon can be distinct things.  The D icon isn't our logo but it does fit style wise with our logo if you see what I mean.

I just don't like the dots between the letters.

File: docs/did-button-branding/index.md (hunk 3 above)
Comment (CrowdHailer) [reply docs/did-button-branding/index.md:13]: better, thanks

**Pull request merged**